sources/packages/backend/libs/services/src/clamav/services/clamav.service.ts

```diff
@@ -22,27 +27,25 @@ export class ClamAVService {
   /**
    * Scans the file for viruses.
    * @param stream stream to be scanned.
-   * @param processSummary process summary logs.
    * @returns boolean true if the stream is virus infected,
    * false if the stream is not virus infected, and
    * null if the stream could not be scanned for viruses.
    */
-  async scanFile(
-    stream: Readable,
-    processSummary: ProcessSummary,
-  ): Promise<boolean> {
+  async scanFile(stream: Readable): Promise<VirusScanCode> {
     try {
       if (!this.scanner) {
         await this.initClam();
       }
       const { isInfected } = await this.scanner.scanStream(stream);
-      return isInfected;
+      return { isInfected };
     } catch (err) {
```

Comment:
The `err` is used as `any` and should be changed to `error: unknown`.
Even though it was added in a previous PR, its usage is now extended.

Reply:
It is no longer a problem since the code was refactored hence closing it.